src/promptflow/promptflow/_cli/_pf/_connection.py

```diff
@@ -16,7 +16,14 @@
 from promptflow._utils.logger_utils import LoggerFactory
 
 logger = LoggerFactory.get_logger(LOGGER_NAME)
-_client = PFClient()
+_client = None
```

Comment:
Please make all tests pass and we can merge the PR

Reply:
done